src/applications/ivc-champva/10-7959C/tests/10-7959C.cypress.spec.js

```diff
@@ -104,35 +124,13 @@ const testConfig = createTestConfig(
     setupPerTest: () => {
       cy.intercept('POST', formConfig.submitUrl, req => {
         cy.get('@testData').then(data => {
-          Object.keys(data).forEach(k => {
-            // Expect all original data used to populate the form
-            // to be present in the submission - this is how we
-            // know that pages we intended to fill didn't get skipped:
-
-            // handle special cases:
-            if (k.endsWith('MedigapPlan')) {
-              // Grab last letter from original data ('medigapPlanK' -> 'K')
-              expect(data[k].slice(-1)).to.equal(req.body[k]);
-            } else if (typeof k === 'object') {
-              // For objects with nested keys, just stringify and check
-              // (easier for things like home address)
-              expect(JSON.stringify(data[k])).to.equal(
-                JSON.stringify(req.body[k]),
-              );
-            } else if (k.includes('DOB') || k.includes('Date')) {
-              // Just check length match. There's a discrepancy in the
-              // format of dates (goes from YYYY-MM-DD to MM-DD-YYYY).
-              // TODO: Address discrepancy at some point.
-              expect(data[k]?.length).to.equal(req.body[k]?.length);
-            } else {
-              expect(data[k] === req.body[k]);
-            }
-          });
+          verifyAllDataWasSubmitted(data, req);
         });
         // Mock the backend response on form submit:
         req.reply({ status: 200 });
       });
       cy.config('includeShadowDom', true);
+      cy.config('retries', { runMode: 0 });
```

Comment:
disables multiple retries for failed E2E tests (this file only). Helps us get to an error faster as otherwise we would have to wait for 3 retries to time-out (which can take quite a while and is not particularly helpful for any of these tests).